ui/app/models/token.js

```diff
@@ -11,6 +11,11 @@ export default class Token extends Model {
   @attr('string') type;
   @hasMany('policy') policies;
   @attr() policyNames;
+  @attr('date') expirationTime;
 
   @alias('id') accessor;
+
+  get isExpired() {
+    return this.expirationTime && this.expirationTime < new Date();
```

Comment:
`question`: `ember-data` sets the date attribute to use `ISO 8601` format whereas JavaScript's `Date` constructor gives us a Unix time. Should we have concern re:  date formatting in this comparison?

Reply:
Hm, this is a good concern to raise but in manual and acceptance testing I have found that the comparison works as expected.